### PR TITLE
Remove fetch for wiki

### PR DESCRIPTION
**Pull request by haslinghuis**

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by blckmn — COMMENTED** [review]
Do we really want to hard code it? It locks us in for ever.

**Comment by haslinghuis** [comment]
Yes all pages are there now. No need to check with fetch anymore as it's makes unneeded network request now.

**Review by blckmn — APPROVED** [review]

**Review by chmelevskij — APPROVED** [review]

**Pull request merged**
